Eshop.Persistence/EshopDbContext.cs

```diff
@@ -14,10 +14,13 @@ public EshopDbContext(DbContextOptions<EshopDbContext> options) : base(options)
         }
 
         public DbSet<Category> Categories  { get; private set; }
+        public DbSet<Product> Products { get; private set; } 
 
-        protected override void OnModelCreating(ModelBuilder builder)
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            builder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new ProductConfiguration());
+            base.OnModelCreating(modelBuilder);
```

Comment:
base is no needed, implementation of the base is empty, this is usually used, when you want extend base, with your override and then call base.